src/bin/vip-config-software-update.js

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import chalk from 'chalk';
+import debugLib from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import command from 'lib/cli/command';
+import { appQuery, appQueryFragments } from 'lib/config/software';
+import { getUpdateResult, promptForUpdate, triggerUpdate } from '../lib/config/software';
+import { ProgressTracker } from '../lib/cli/progress';
+import UserError from '../lib/user-error';
+
+const debug = debugLib( '@automattic/vip:bin:config-software' );
+
+const UPDATE_SOFTWARE_PROGRESS_STEPS = [
+	{ id: 'trigger', name: 'Trigger software update' },
+	{ id: 'process', name: 'Process software update' },
+];
```

Comment:
Do we need `success` and `error` here?

Reply:
What do you mean? The proxyTracker has a way to mark each step as success / error (it changes to :heavy_check_mark: or :x: in the print out)